### ISSUE TITLE
Caption error

### ISSUE THREAD

**Issue by FuchsFuchs1** [esmBot/esmBot#251]
### Command that caused the error

Caption

### Image that caused the error

https://tenor.com/view/error-windows-windows-error-message-windows-application-error-gif-17157659

### Steps to reproduce

1. post this link in direct messages _https://tenor.com/view/error-windows-windows-error-message-windows-application-error-gif-17157659_
2. run this cmd 
_caption The AI that took over the world after i uploaded the hol Rule34 catalog to theyer Database_

### Self-hosted instance?

No

### Error file

```
Message: TypeError: Cannot read properties of undefined (reading 'has')
    at CaptionCommand.run (file:///home/esmBot/esmBot/classes/imageCommand.js:115:45)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async default (file:///home/esmBot/esmBot/events/messageCreate.js:105:20)

Stack Trace: TypeError: Cannot read properties of undefined (reading 'has')
    at CaptionCommand.run (file:///home/esmBot/esmBot/classes/imageCommand.js:115:45)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async default (file:///home/esmBot/esmBot/events/messageCreate.js:105:20)
```

**Comment by bjcscat** [comment]
permissions error with dms

**Comment by TheEssem** [comment]
Should be fixed with a438fe26df509c6fda6e0b2bc09af01cadb51715
